src/import-utils.ts

```diff
@@ -11,25 +11,48 @@
  * subject to an additional IP rights grant found at
  * http://polymer.github.io/PATENTS.txt
  */
-import * as path from 'path';
-import * as urlLib from 'url';
-const pathPosix = path.posix;
 import * as dom5 from 'dom5';
-import encodeString from './third_party/UglifyJS2/encode-string';
-
 import * as parse5 from 'parse5';
 import {ASTNode} from 'parse5';
-import constants from './constants';
+import * as urlLib from 'url';
+
 import * as astUtils from './ast-utils';
+import {AssignedBundle, BundleManifest} from './bundle-manifest';
+import constants from './constants';
 import * as matchers from './matchers';
+import encodeString from './third_party/UglifyJS2/encode-string';
 import * as urlUtils from './url-utils';
-import {AssignedBundle, BundleManifest} from './bundle-manifest';
 import {UrlString} from './url-utils';
 
 // TODO(usergenic): Revisit the organization of this module and *consider*
 // building a class to encapsulate the common document details like docUrl and
 // docBundle and global notions like manifest etc.
 
+/*
+ * Given an import document with a base tag, transform all of its URLs and set
+ * link and form target attributes and remove the base tag.
+ */
+export async function debaseDocument(docUrl: UrlString, importDoc: ASTNode) {
```

Comment:
Recommend removing the async here. Callers don't seem to be awaiting on it, and it doesn't have any awaits itself